### PR TITLE
no default file service

### PR DESCRIPTION
**Pull request by reusee**
## What type of PR is this?

- [ ] API-change
- [ ] BUG
- [x] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #

## What this PR does / why we need it:
It's definitely wrong to use the default file service.

### CODE REVIEW

File: pkg/testutil/util_new.go [hunk 1]
```diff
@@ -92,7 +92,7 @@ func NewFS() *fileservice.FileServices {
 		panic(err)
 	}
 	fs, err := fileservice.NewFileServices(
-		"local",
+		"",
```
Comment (fengttt) [inline pkg/testutil/util_new.go:95]: why we replace "local" with ""?   Seems "local" is perfectly ok name, better than "".

File: pkg/tnservice/store_test.go [hunk 2]
```diff
@@ -108,7 +108,7 @@ func TestStartWithReplicas(t *testing.T) {
 			return nil, err
 		}
 		return fileservice.NewFileServices(
-			defines.LocalFileServiceName,
+			"",
```
Comment (fengttt) [inline pkg/tnservice/store_test.go:111]: Here

File: pkg/testutil/util_new.go (hunk 1 above)
Comment (reusee) [reply pkg/testutil/util_new.go:95]: "" means no default, and will panic if use the FileServices instance without Get.

File: pkg/tnservice/store_test.go (hunk 2 above)
Comment (reusee) [reply pkg/tnservice/store_test.go:111]: the first argument of NewFileServices is the default service name. We're removing the default file service.

**Comment by reusee** [comment]
I guess the changes expose something wrong in the codebase, I will take a look after dealing with some other issues.

**Review by zhangxu19830126 — APPROVED** [review]

**Review by xzxiong — APPROVED** [review]

**Review by nnsgmsone — APPROVED** [review]

**Review by fengttt — APPROVED** [review]

**Comment by reusee** [comment]
@mergifyio refresh

**Pull request merged**